src/remove_chars.py

```diff
@@ -0,0 +1,9 @@
+"""Remove the first and last char from a string."""
+
+
+def remove_char(s):
+    """Remove the first and last char from a string."""
+    a = list(s)
```

Comment:
Another good solution is using index slicing, which is available on any indexable data type, e.g. lists, strings and tuples. If you don't know about index slicing, you definitely need to! Look it up in the Python docs.